ftests/core/src/main/java/org/commonjava/indy/data/DefaultStoreValidatorTest.java

```diff
@@ -106,21 +106,53 @@ public void run() throws Exception {
             new RemoteRepository( "maven", "validation-indy-allowed", "http://127.0.0.1" );
 
         LOGGER.warn("=> Start Validating RemoteRepository: [" + allowedRemoteRepo.getUrl()+"]");
-        RemoteRepository remoteRepository3 = client.stores().create(allowedRemoteRepo, changelog,
-            RemoteRepository.class);
+        RemoteRepository remoteRepository3 = client.stores().create(allowedRemoteRepo, changelog, RemoteRepository.class);
 
 
         ArtifactStoreValidateData validateAllowedRepo = validator.validate(remoteRepository3);
-        LOGGER.warn("=> Returned [Allowed Not Valid ( !GET | HTTP ) SSL] ArtifactStoreValidateData: " + validateAllowedRepo.toString());
+        LOGGER.warn("=> Returned [Allowed Not Valid ( GET | HTTP ) SSL] ArtifactStoreValidateData: " + validateAllowedRepo.toString());
 
         assertNotNull( validateAllowedRepo );
         assertFalse(validateAllowedRepo.isValid());
-        assertNull(validateAllowedRepo.getErrors().get("HTTP_GET_STATUS"));
-        assertNull(validateAllowedRepo.getErrors().get("HTTP_HEAD_STATUS"));
+        assertNotNull(validateAllowedRepo.getErrors().get("HTTP_GET_STATUS"));
+        assertNotNull(validateAllowedRepo.getErrors().get("HTTP_HEAD_STATUS"));
         assertNotNull( validateAllowedRepo.getErrors().get("disabled") );
 
 
 
+        RemoteRepository npmRemoteRepo =
+          new RemoteRepository( "npm", "validation-indy-npm", "https://repository.engineering.redhat.com/nexus/" );
```

Comment:
We cannot put this in a test in our codebase, sorry

Reply:
you are right, i was just wondering why this npm repo was disabled. i will remove foreign urls from tests